stable/nginx-ingress/README.md

```diff
@@ -53,6 +53,7 @@ Parameter | Description | Default
 `controller.defaultBackendService` | default 404 backend service; required only if `defaultBackend.enabled = false` | `""`
 `controller.scope.enabled` | limit the scope of the ingress controller | `false` (watch all namespaces)
 `controller.scope.namespace` | namespace to watch for ingress | `""` (use the release namespace)
+`controller.serviceAccount` | Service account to run under | `default`
```

Comment:
Since (as @prydonius noted) the field's name is `serviceAccountName`, let's use that as the key name.